### PR TITLE
#170947535 Eslint Configuration

### PR DESCRIPTION
**Pull request by Baraka-Mugisha**
#### What does this PR do?
- Configures Eslint

#### Description of Task to be completed?
- install eslint
- add eslint rules

#### How should this be manually tested?
- clone repo
- do npm install

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#170947535](https://www.pivotaltracker.com/story/show/170947535)

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  }],
+  "require-jsdoc": ["error", {
+      "require": {
+          "FunctionDeclaration": true,
+          "MethodDefinition": true,
+          "ClassDeclaration": true
+      }
+  }]
+}
+}
```
Comment (Mnickii) [inline .eslintrc.json]: add newline here; configure your editor to automatically do this

**Review by Mnickii — DISMISSED** [review]
Approved, but rebase this PR

**Review by Ntare22 — DISMISSED** [review]
Please rebase

**Comment by Baraka-Mugisha** [comment]
The PR is now rebased

**Review by bbaime98 — APPROVED** [review]

**Review by Ntare22 — APPROVED** [review]

**Review by Mnickii — APPROVED** [review]

**Pull request merged**